recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
```

Comment:
> some git repos can become very huge

You have a point there. Not sure how conda clones the repository (what is the depth of the clone?). I don't like to use GitHub tagged files though. Users can delete them and re-add a new one with the same tag but fixing a problem, and then you have a checksum issue on your hands. PyPI solved that by forcing people to bump the version because PyPI does not accept the same file name/version for the same release even if you delete it.

(It would be nice to have `pygridtools` source listed at PyPI and then I would use the file instead. @phobson interested?)